tools/diff.js

```diff
@@ -17,7 +17,7 @@ async function main() {
         }
     }
     for (let item of attemptedSet2) {
-        recipe = item.split(",,").map(x => data1.icons[x] + " " + x)
```

Comment:
Is this so we utilize the most up-to-date icons?

Reply:
the second file can have emojis that are not in the first file, so when looking up an emoji using a recipe from file 2, the emoji should come from file 2, or else the diff output has a bunch of `undefined` where the emoji should be